external_search.py

```diff
@@ -47,12 +52,20 @@ def __init__(self, _db, url=None, works_index=None):
                     goal=ExternalIntegration.SEARCH_GOAL
                 )
 
+                if not integration:
+                    raise CannotLoadConfiguration(
+                        "No Elasticsearch integration configured."
+                    )
```

Comment:
Without the integration configured I was getting an unhelpful message shortly below this point.